lib/diagnostic.js

```diff
@@ -39,20 +51,39 @@ const owner = null
 // Methods: vocalize
 
 // Replace null with your answer
-const pet = null
+const pet = {
+  name: 'Buster',
+  species: 'cat',
+  breed: 'unknown',
+  vocalization: 'meows',
+  vocalize: function() { return console.log(this.vocalization)}
```

Comment:
You can use either just `console.log(this.vocalization)` or `return(this.vocalization)`